app/lib/summary_document_generator.rb

```diff
@@ -0,0 +1,23 @@
+class SummaryDocumentGenerator
```

Comment:
I'm probably being a little picky but this isn't actually *generating* a summary document. Maybe `SummaryDocumentUrl` is more suited?